cri/v1alpha2/cri_utils_test.go

```diff
@@ -1854,3 +1855,50 @@ func Test_toCNIPortMappings(t *testing.T) {
 		})
 	}
 }
+
+// CRI test: apply container config by annotation
+func Test_applyContainerConfigByAnnotation(t *testing.T) {
```

Comment:
The name of this function is `applyContainerConfigByAnnotation`. It means there would be other annotation follows, not just `resource.memory_swap`. 
I think we should make the test more generally for the coming annotation, change the parameters to `annotation map[string]string`